src/redux/books/booksSlice.js

```diff
@@ -1,19 +1,93 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
 import { v4 as uuidv4 } from 'uuid';
-import bookItems from '../bookItems';
+// import bookItems from '../bookItems';
+// import thunk from 'redux-thunk';
 
```

Comment:
- [ ] Please, let's try not to have commented unused code in our codebase as this makes our code not neat and not professional. It's a good practice to not have commented unused codes in our codebase. Kindly remove it 👍🏽